Makefile

```diff
@@ -5,11 +5,16 @@ export
 STEPS := all tests apps clean check info tools %.debug
 .PHONY: $(STEPS) %.debug shell docker
 
+NPROC := nproc
+ifeq ($(HOST),Darwin)
+  NPROC := sysctl -n hw.physicalcpu
+endif
+
 CWD := $(shell pwd)
 HOST := $(shell uname)
```

Comment:
The `NPROC` check is using `HOST ` when `HOST ` is declared after. Please move the `HOST` check before `NPROC`

Reply:
Done.